kivy/app.py

```diff
@@ -982,6 +991,17 @@ def stop(self, *largs):
                 self._app_window.remove_widget(child)
         App._running_app = None
 
+    def pause(self, *largs):
+        '''Pause the application.
+
+        On Android set OS state to pause, Kivy app state follows.
+        No functionality on other OS.
+        .. versionadded:: 2.2.0
+        '''
+        if platform == 'android':
+            from android import mActivity
+            mActivity.moveTaskToBack(True)
```

Comment:
Minor: Maybe we can add a log that warns the developer about the non-availability?

Reply:
Done